### DISCUSSION TITLE
Nonlocal modeling to capture grain refinement under dislocation variations and slip band formation

### DISCUSSION THREAD

**Discussion by maziartour** [damask-multiphysics/DAMASK#15] [Q&A]
Hello,
I am trying the nonlocal modeling in DAMASK (“nonlocal_Ni” from DAMASK documentation). I have attached the related files to this message However, the model cannot obtain new sub-grain low-angle grain boundary line formation inside grains with slip localization and dislocation accumulations during deformation to match my experimental data. I need to simulate grain texture variations and grain refinement to match in-situ EBSD data. Can you tell me if there is such model in DAMASK? 

[Nnlocl_200-200-1.zip](https://github.com/user-attachments/files/16076216/Nnlocl_200-200-1.zip)

**Comment by MarDiehl** [comment]
Probably DAMASK is not suitable for your needs.
